Source/Util/PlayerRetriever.cs

```diff
@@ -0,0 +1,23 @@
+﻿using GameNetcodeStuff;
+
+namespace PlayerDogModel_Plus.Source.Util
+{
+    internal class PlayerRetriever
+    {
+        internal static PlayerControllerB GetPlayerFromClientId(ulong clientId)
+        {
+            if (StartOfRound.Instance.ClientPlayerList.TryGetValue(clientId, out int playerScriptIndex))
```

Comment:
90% of the time, this logic works to grab a `PlayerControllerB` instance. However, when a lobby is remade, then this can fail.